templates/default/consul/nm-consul.hcl.erb

```diff
@@ -42,5 +42,17 @@ services = [
             id = "http-health-check"
             alias_service = "nm/rpc"
         }
+    },
+    {
+        id = "nm/monitoring"
+        name = "nodemanager"
+        tags = [
+            "monitoring"
+        ]
+        port = <%= node['hops']['nm']['metrics_port'] %> 
+        check = {
+            id = "http-health-check"
```

Comment:
Wrong check id. It should be unique at the machine

Reply:
fixed